Winbox-mac.app/Contents/MacOS/winbox-mac

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env arch -x86_64 bash
```

Comment:
Can you explain this change please?

Reply:
`#!/usr/bin/env` allows us to run any binary in the users `$PATH`,`arch -x86_64` forces `bash` to launch the Intel x86_64 slice.

Reply:
Thanks for explanation.